### PR TITLE
2829 review club

### PR DESCRIPTION
**Pull request by jbesraa**

### CODE REVIEW

**Comment by dunxen** [comment]
Nice. I like the addition of the bLIP review. I do think we could expand on the questions to cover more detail and specifics, though. I'll review and give them some thought. If you have any additional specific questions in mind then it would be great to add some! 

We could dive deeper into the implementation itself.

**Comment by jbesraa** [comment]
> Nice. I like the addition of the bLIP review. I do think we could expand on the questions to cover more detail and specifics, though. I'll review and give them some thought. If you have any additional specific questions in mind then it would be great to add some!
> 
> We could dive deeper into the implementation itself.

Added another question, and I have a couple of follow up questions if the audience will be engaging enough.
Notice that this sessions is a first of two and in the second one we will dive into the [pr](https://github.com/lightningdevkit/rust-lightning/pull/2829)

File: _posts/2024-02-16-#2829.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: pr
+date: 2024-02-16
```
Comment (jkczyz) [inline _posts/2024-02-16-#2829.md:3]: Looks like this needs to be update.

File: _posts/2024-02-16-#2829.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+## Notes
+  This is part one of the review for [bLIP/31] named "Mutual Message Exchange".
+  In this part, we will review the bLIP, and in the next session we will review
+  the implementation at [ldk/2829].
+  
+
+## Questions
+1. Did you review bLIP31? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/lightningdevkit/rust-lightning/blob/master/CONTRIBUTING.md#peer-review)?
+2. What are the capabilities [bLIP/31] introduce?
+3. Who initiae the messaging process? what are the pros/cons?
```
Comment (jkczyz) [inline _posts/2024-02-16-#2829.md:22]: s/initiae/initiates

Comment (jkczyz) [inline _posts/2024-02-16-#2829.md:22]: s/what/What

File: _posts/2024-02-16-#2829.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+4. How does [bLIP/31] relate to [bolt/11] and [bolt/12]?
+5. How the message sender know they are able to exchange a message with the recipient?
```
Comment (jkczyz) [inline _posts/2024-02-16-#2829.md:24]: s/How/How does

**Comment by jbesraa** [comment]
@jkczyz thanks!
 addressed all the points

**Review by dunxen — APPROVED** [review]

**Comment by jbesraa** [comment]
Ah no worries! Thanks for merging. In the next few days ill open part 2 of this. 

In the next couple of sessions after that, gonna cover the new dns bLIP.

**Pull request merged**